canonical-web-and-design/maas-ui
# Fetch data required for machines list.

## Done
- Fetched data required for building machine list with double row.

## QA
- Go the new machine list.
- Check that a loading spinner shows while machines load.
- Check that a bunch of things are fetched and show up in redux, and those things match what are fetched in the angular machines list (check Network tab of developer tools).
